Implement reform of gesetzliche Pflegepflegeversicherung effective as of 2023-07-01 (#593)
iza-institute-of-labor-economics/gettsim: docs/gettsim_objects/input_variables.md

Tax Bracket (1 to 5) for withholding tax

Type: int

> ## `anz_kinder_u_25`

Reviewer: ```suggestion
## `eigene_kind_bis_24`
```

> Amount of children below tha age of 25

Reviewer: ```suggestion
Amount of own children below the age of 25 (incl. not in hh)
```

I assume not in hh count as, well? Otherwise, we won't need a new input variable.

> Amount of own children below the age of 25 (incl. not in hh)

Reviewer: ```suggestion
Number of own children below the age of 25 (incl. not in hh)
```
